api/v1/model_handlers.go

```diff
@@ -57,6 +57,13 @@ func (a *Api) RegisterModel(container *restful.Container) {
 		Doc("Get a list of all available metrics for the Cluster entity").
 		Operation("availableMetrics"))
 
+	// The /stats/ endpoint returns a list of all available stats for the Cluster entity of the model.
+	ws.Route(ws.GET("/stats/").
+		To(a.clusterStats).
+		Filter(compressionFilter).
+		Doc("Get all available stats for the Cluster entity").
+		Operation("clusterStats"))
```

Comment:
Aren't all of these new endpoints missing `Writes()`?

Also, many other endpoints say `Writes(types.MetricResult{})` but that's just wrong, e.g. `GET("/nodes/")`